demisto_sdk/commands/unify/tests/unifier_test.py

```diff
@@ -206,12 +234,244 @@ def test_insert_script_to_yml_exceptions(package_path, dir_name, file_path):
+        self.export_dir_path = os.path.join(self.test_dir_path, self.package_name)
+        self.expected_yml_path = os.path.join(self.test_dir_path, 'integration-SampleIntegPackage.yml')
+
+    def teardown(self):
+        if self.test_dir_path:
+            shutil.rmtree(self.test_dir_path)
+
+    def test_unify_integration(self):
+        """
+        sanity test of merge_script_package_to_yml of integration
+        """
+        from demisto_sdk.commands.unify.unifier import Unifier
+
+        create_test_package(
+            test_dir=self.test_dir_path,
+            package_name=self.package_name,
+            base_yml='demisto_sdk/tests/test_files/Unifier/SampleIntegPackage/SampleIntegPackage.yml',
+            script_code=TEST_VALID_CODE,
+            detailed_description=TEST_VALID_DETAILED_DESCRIPTION,
+            image_file='demisto_sdk/tests/test_files/Unifier/SampleIntegPackage/SampleIntegPackage_image.png',
+        )
+
+        unifier = Unifier(indir=self.export_dir_path, outdir=self.test_dir_path)
+        yml_files, orig_yml, orig_script, orig_image, orig_description = unifier.merge_script_package_to_yml()
+        export_yml_path = yml_files[0]
+
+        assert export_yml_path == self.expected_yml_path
+        assert orig_yml == f'{self.export_dir_path}/SampleIntegPackage.yml'
+        assert orig_script == f'{self.export_dir_path}/SampleIntegPackage.py'
+        assert orig_image == f'{self.export_dir_path}/SampleIntegPackage_image.png'
+        assert orig_description == f'{self.export_dir_path}/SampleIntegPackage_description.md'
+
+        comment = '# this is a comment text inside a file 033dab25fd9655480dbec3a4c579a0e6'
+        with open(export_yml_path) as file_:
+            unified_content = file_.read()
+        assert comment in unified_content
+
+        actual_yml = get_yaml(export_yml_path)
+
+        expected_yml = get_yaml('demisto_sdk/tests/test_files/Unifier/SampleIntegPackage/'
+                                'integration-SampleIntegPackageSanity.yml')
+
+        assert expected_yml == actual_yml
+
+    def test_unify_integration__detailed_description_with_special_char(self):
+        """
+        -
+        """
+        from demisto_sdk.commands.unify.unifier import Unifier
+        description = '''
+        some test with special chars
+        שלום
+        hello
+        你好
+        '''
+
+        create_test_package(
+            test_dir=self.test_dir_path,
+            package_name=self.package_name,
+            base_yml='demisto_sdk/tests/test_files/Unifier/SampleIntegPackage/SampleIntegPackage.yml',
+            script_code=TEST_VALID_CODE,
+            image_file='demisto_sdk/tests/test_files/Unifier/SampleIntegPackage/SampleIntegPackage_image.png',
+            detailed_description=description,
+        )
+
+        unifier = Unifier(self.export_dir_path, outdir=self.test_dir_path)
+        yml_files, orig_yml, orig_script, orig_image, orig_description = unifier.merge_script_package_to_yml()
```

Comment:
what is the reason that `merge_script_package_to_yml()` returns all these file pathes?
shouldn't it just return the path to the generated yml?

Reply:
idk, did not want to break something that depends on it (i'm assuming no one actually need them.
should i remove them?

Reply:
no need for them, remove them. Return just the path to the unified yml, and make sure that yml is created as expected